### PR TITLE
Feat/strategy landing pages

### PR DESCRIPTION
**Pull request by codingki**
## Description

This PR is for Strategy landing pages 

## Changes

- [x] added strategies landing pages for clear gate cellars
- [x] removed `/pages/cellars/` -> `/pages/strategies/`
- [x] changed flow cellar card -> strategy landing page -> choose wanna manage on somm or exchange
- [x] moved `cellars/[id]` ->  `strategies/[id]/manage`

## Screenshots (if appropriate):
image

### CODE REVIEW

**Comment by xanderjl** [comment]
image

Looks like we have an import path mismatch

**Comment by codingki** [comment]
@xanderjl just pushed a fix in 
[2d0201b](https://github.com/strangelove-ventures/sommelier/pull/546/commits/2d0201b07b6e6971d5f5e287569d3190d6da1587)

**Review by joshuanatanielnm — APPROVED** [review]
LGTM

File: src/data/strategyPageContentData.tsx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    <br/><br/>
+    The strategy is more sensitive to positive market movements than a trend-following strategy and is expected to provide slightly higher returns in a bull market than a similar trend-following strategy but lower returns in a bear market.`,
+    backtestingImage: "/assets/images/btc-eth-moment-backtesting.png",
+    backtestingText: `Notes: Performance of the strategy from Jan 2019-Jun 2022. Black line for benchmark
+
+      Backtest results:
+      <div style="display:flex;">
```
[the remark this answers is not in the document]
Comment (joshuanatanielnm) [reply src/data/strategyPageContentData.tsx:131]: Okay sure

**Review by xanderjl — APPROVED** [review]
I think this could've been broken up into a few PRs- route refactor and then maybe landing page components could've been separate for example. Other than that, this is stellar work y'all! Send it! 💪 🚀

**Pull request merged**